appintro/src/main/java/com/github/appintro/AppIntroBase.kt

```diff
@@ -156,17 +156,36 @@ abstract class AppIntroBase : AppCompatActivity(), AppIntroViewPagerListener {
         }
     }
 
+
+    /**
+     * Adds a new slide at the end of the Intro and associates the permissions with it
+     * @param fragment Instance of Fragment which should be added as slide.
+     * @param permissions - Array of permissions
+     * @param required - Whether the user can change this slide without granting the permissions.
+     */
+    protected fun addSlideWithPermissionRequest(fragment: Fragment, permissions: Array<String>, required: Boolean = true) {
```

Comment:
```suggestion
    protected fun askForPermissions(permissions: Array<String>, slide: Fragment, required: Boolean = true)
```

Reply:
I dont quite get what you are trying to do here, this duplicates the askForPermissions-function name and parameters, which doesn't compile?

Do you want me to change the function name or order of parameters?

Reply:
Sorry I haven't seen you added the overload already. This function can go then

Reply:
It is only a tiny wrapper but one that, i imagine, will be useful to most people, since it takes care of the "combining of the permission and slide". Should i really remove it?

Reply:
I think we really don't need an extra api where this can be achieved with 2 statements.

Reply:
Done!